singTest.sh

```diff
@@ -49,3 +49,8 @@ set -x
 ${SING_EXE} manifest -a $a -f manifest.txt
 set +x 
 cat manifest.txt
+
+header "Suppressing Analysis"
```

Comment:
Remove this as well. We don't need this script to test everything, plus its suppressing something that wasn't even published.